index.js

```diff
@@ -37,7 +37,8 @@ module.exports = {
             singleQuote: true,
             semi: false,
             tabWidth: 2,
-            trailingComma: "none",
+            printWidth: 80,
```

Comment:
It seems you sneaked this option too, what does it affect?

Reply:
It actually does nothing, since it's the default (see the diffs for the rapu/frontend PR's; no line lengths changed).

But I thought since we have thousands upon thousands of lines formatted according to these rules, maybe it's good to include the value here. 🤷‍♂️